wl-paste: add page, wl-copy: update link (#7123)
tldr-pages/tldr: pages/linux/wl-paste.md

# wl-paste

> Tool to access data stored in the clipboard for Wayland.
> See also: `wl-copy`.

> > More information: <https://manned.org/wl-paste>.

Reviewer: We should use the official homepage here. This probably should also be updated for `wl-copy`.
```suggestion
> More information: <https://github.com/bugaevc/wl-clipboard>.
```

> - Output the contents of the clipboard:

Reviewer: ```suggestion
- Paste the contents of the clipboard:
```

`wl-paste`

> - Paste the contents of the clipboard once then clear the clipboard:

Reviewer: ```suggestion
- Paste the contents of the clipboard and then clear it:
```

`wl-paste --paste-once`

> - Paste the contents of the clipboard to a file:

Reviewer: ```suggestion
- Write the contents of the clipboard to a file:
```
